projects/1.19/assets/create-dreams-desires/create_dd/lang/zh_cn.json

```
@@ -0,0 +1,702 @@
+  "block.create_dd.spirit_trapdoor": "灵魂木活板门",
+  "block.create_dd.spirit_pressure_plate": "灵魂木压力板",
+  "block.create_dd.spirit_button": "灵魂木按钮",
+  "block.create_dd.rubber_log": "橡胶原木",
+  "block.create_dd.rubber_leaves": "橡胶树叶",
+  "block.create_dd.rubber_sapling": "橡胶树苗",
+  "block.create_dd.stripped_rubber_log": "去皮橡胶原木",
+  "block.create_dd.rubber_wood": "橡胶木",
+  "block.create_dd.stripped_rubber_wood": "去皮橡胶木",
+  "block.create_dd.rubber_planks": "橡胶木板",
+  "block.create_dd.rubber_stairs": "橡胶楼梯",
+  "block.create_dd.rubber_slab": "橡胶台阶",
+  "block.create_dd.rubber_fence": "橡胶木栅栏",
+  "block.create_dd.rubber_fence_gate": "橡胶木栅栏门",
+  "block.create_dd.rubber_door": "橡胶木门",
+  "block.create_dd.rubber_trapdoor": "橡胶木活板门",
+  "block.create_dd.rubber_pressure_plate": "橡胶木压力板",
+  "block.create_dd.rubber_button": "橡胶木按钮",
+  "block.create_dd.train_scaffolding": "列车脚手架",
+  "block.create_dd.hydraulic_scaffolding": "液压脚手架",
+  "block.create_dd.overburden_scaffolding": "板固脚手架",
+  "block.create_dd.industrial_scaffolding": "工业脚手架",
+  "block.create_dd.mithril_scaffolding": "秘银脚手架",
+  "block.create_dd.mossy_andesite_scaffolding": "覆苔安山脚手架",
+  "block.create_dd.zinc_scaffolding": "锌脚手架",
+  "block.create_dd.tin_scaffolding": "锡脚手架",
+  "block.create_dd.steel_scaffolding": "钢脚手架",
+  "block.create_dd.netherite_scaffolding": "下界合金脚手架",
+  "block.create_dd.bronze_scaffolding": "青铜脚手架",
+  "block.create_dd.shadow_steel_scaffolding": "暗影脚手架",
+  "block.create_dd.refined_radiance_scaffolding": "光辉脚手架",
+  "block.create_dd.blaze_gold_scaffolding": "§e烈焰黄铜脚手架",
+  "block.create_dd.stargaze_singularity_scaffolding": "§e星幻奇点脚手架",
+  "block.create_dd.overcharged_scaffolding": "§e过充合金脚手架",
+  "block.create_dd.vertical_framed_split_glass": "垂直分隔镶框玻璃",
+  "block.create_dd.horizontal_framed_split_glass": "水平分隔镶框玻璃",
+  "block.create_dd.framed_split_glass": "分隔镶框玻璃",
+  "block.create_dd.ornate_iron_glass": "华丽铁玻璃",
+  "block.create_dd.vertical_framed_split_glass_pane": "垂直分隔镶框玻璃板",
+  "block.create_dd.horizontal_framed_split_glass_pane": "水平分隔镶框玻璃板",
+  "block.create_dd.framed_split_glass_pane": "分隔镶框玻璃板",
+  "block.create_dd.ornate_iron_glass_pane": "华丽铁玻璃板",
+  "block.create_dd.copycat_block": "伪装方块",
+  "block.create_dd.copycat_slab": "伪装台阶",
+  "block.create_dd.spectral_ruby_lamp": "光灵红宝石灯",
+  "block.create_dd.reversed_gearshift": "反转齿轮箱",
+  "block.create_dd.hydraulic_press": "液压机",
+  "block.create_dd.bronze_drill": "青铜钻头",
+  "block.create_dd.radiant_drill": "光辉钻头",
+  "block.create_dd.shadow_drill": "暗影钻头",
+  "block.create_dd.radiant_panel": "光辉能量板",
+  "block.create_dd.shadow_panel": "暗影能量板",
+  "block.create_dd.bronze_saw": "青铜锯",
+  "block.create_dd.bronze_bell": "青铜钟",
+  "block.create_dd.2_blade_fan": "大型风扇（2）",
+  "block.create_dd.4_blade_fan": "大型风扇（4）",
+  "block.create_dd.8_blade_fan": "大型风扇（8）",
+  "block.create_dd.secondary_encased_chain_drive": "二级链式传动箱",
+  "block.create_dd.secondary_adjustable_chain_gearshift": "二级可调节链式传动箱",
```

根据机械动力原版翻译，这是二号而不是二级，如“轴向底盘二号”
```suggestion
  "block.create_dd.secondary_encased_chain_drive": "二级链式传动箱",
  "block.create_dd.secondary_adjustable_chain_gearshift": "二级可调节链式传动箱",
```